guides/common/modules/proc_installing-packages-on-a-host.adoc

```diff
@@ -0,0 +1,15 @@
+[id="Installing_Packages_on_a_Host_{context}"]
+= Installing Packages on a Host
+
+Use this procedure to review and install packages on a host using the {ProjectWebUI}.
+The list of packages available for installation depends on the Content View and Lifecycle Environment assigned to the host.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and select the host you want to install packages on.
+. On the *Content* tab, click *Packages* > click the vertical ellipsis icon next to the upgrade button, and select *Install Packages*.
```

Comment:
```suggestion
. On the *Content* tab, select the *Packages* tab.
. On the vertical ellipsis icon next to the upgrade button, click *Install Packages*.
```

Reply:
This definitely looks way better! Thanks!